### ISSUE TITLE
EJBCA container stops right after docker start

### ISSUE THREAD

**Issue by sandevins** [bitnami/bitnami-docker-ejbca#10]
Hi,

Yesterday I have installed the two containers manually (ejbca and mariadb) and everything worked fine. Now I just started my PC and tries to start the containers (first the mariadb container and second the ejbca container). The ejbca one stops right after the docker start.

The result of docker logs is the following.

```
ejbca 06:52:12.09 Welcome to the Bitnami ejbca container
ejbca 06:52:12.09 Subscribe to project updates by watching https://github.com/bitnami/bitnami-docker-ejbca
ejbca 06:52:12.09 Submit issues and feature requests at https://github.com/bitnami/bitnami-docker-ejbca/issues
ejbca 06:52:12.09 
ejbca 06:52:12.09 INFO  ==> ** Starting ejbca setup **
ejbca 06:52:12.10 INFO  ==> Validating settings in EJBCA_* env vars...
ejbca 06:52:12.10 INFO  ==> Initializing EJBCA...
ejbca 06:52:12.10 INFO  ==> Persisted data detected
ejbca 06:52:12.11 INFO  ==> Loading persisted keystore passwords
ejbca 06:52:12.11 INFO  ==> Placing widlfly keystores
```

Is there any special way to start the container?

Best Regards

**Comment by sandevins** [comment]
Hi,

Did you try to do a docker stop ejbca followed by a docker start ejbca?

Best regards,

**Comment by carrodher** [comment]
You're right, I was able to reproduce the issue when stopping/starting the container.
I just created an internal task to investigate and fix this issue. I'm adding the `on-hold` label to prevent the stale bot from closing the issue while we evaluate it.

I can't provide an ETA about when this task is going to be fully done.

**Comment by carrodher** [comment]
Hi, I want to share some news:
A new revision of the image was released (`7.4.3-2-debian-10-r68`), see the [notable changes](https://github.com/bitnami/bitnami-docker-ejbca#743-2-debian-10-r68) in the _README.md_ and the [commit](https://github.com/bitnami/bitnami-docker-ejbca/commit/61afc059be1967ab8347ad371209b24f6d6c68f8) where changes were implemented.

I'm closing this issue as the original problem should be solved, please don't hesitate to reopen it or create a new one if you detect a weird behavior.
